pom.xml

```diff
@@ -336,6 +336,16 @@
 				<artifactId>hapi-base</artifactId>
 				<version>2.0</version>
 			</dependency>
+			<dependency>
+				<groupId>ca.uhn.hapi</groupId>
+				<artifactId>hapi-structures-v23</artifactId>
+				<version>2.0</version>
```

Comment:
Are these dependencies necessary for adding the tests?

Reply:
@dkayiwa I will make the neccessary changes. Thanks. Please give feedback on #2479